src/breakpoints.jl

```diff
@@ -193,9 +193,6 @@ function prepare_slotfunction(framecode::FrameCode, body::Union{Symbol,Expr})
     assignments = Expr[:($dataname = $framename.framedata)]
     default = Unassigned()
     for slotname in framecode.src.slotnames
```

Comment:
I guess `uslotnames` could be removed and make the loop go over `unique(framecode.src.slotnames)` instead?

Reply:
Ye.